src/i18n/it_IT/vehicles.ts

```diff
@@ -606,4 +606,310 @@ export default {
         icon: 'car-side',
         possibleBuildings: [6, 19],
     },
+    42: {
+        caption: 'Auto della polizia locale',
+        color: '#0a2bff',
+        credits: -1,
```

Comment:
-1 doesnt seem correct

Reply:
> -1 doesnt seem correct

I managed to add the credits and coins for the purchase of the vehicle:
42 - 43 - 44 - 45
For the others from 46 onwards, I don't have the suitable buildings, and I can't verify how much they cost